components/Tabs.tsx

```diff
@@ -0,0 +1,123 @@
+import { Tab as HeadlessTab } from '@headlessui/react'
```

Comment:
Do we need this file to be imported? I see in the MDX file you're importing tabs directly from nextra (which is right!)

Reply:
Huh, tabs are working without this file now. 
When I tried before I was getting some crazy errors. I wonder if that was a nextra error.
Anyways, I took it out.